src/sources/kubernetes_logs/mod.rs

```diff
@@ -368,6 +396,14 @@ fn default_self_node_name_env_template() -> String {
     format!("${{{}}}", SELF_NODE_NAME_ENV_KEY.to_owned())
 }
 
+fn default_max_read_bytes() -> usize {
+    2048
```

Comment:
Did we determine if raising this helps?

Reply:
Raising this didn't affect performance a lot in my testing.